### PR TITLE
ci: use new bee-factory

### PR DESCRIPTION
**Pull request by vojtechsimetka**

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -9,21 +9,16 @@ on:
...
+  BEE_DEBUG_API_URL: 'http://127.0.0.1:1635'
 
 jobs:
   nodejs:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 17.x]
```
Comment (AuHau) [inline .github/workflows/tests.yaml:21]: What about adding Node 18?

**Review by AuHau — CHANGES_REQUESTED** [review]
This won't work. You have to have somewhere specified the Bee version you want to run. I know you bee-js as example but there the Bee version is specified in the `packege.json -> engines.bee`, where Bee Factory looks for it as described at https://github.com/ethersphere/bee-factory/#external-bee-version-configuration

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (vojtechsimetka) [reply .github/workflows/tests.yaml:21]: Even on bee-js we don't test against node v 18. I tried it but te orchestraction fails

**Review by AuHau — APPROVED** [review]
LGTM

**Pull request merged**